### DISCUSSION TITLE
Inferencing on Fine Tuned Mistral 7B

### DISCUSSION THREAD

**Discussion by Manivarsh-adi** [vllm-project/vllm#2085] [General]
Hello,

I have fine tuned mIstral 7B on a specific data and want to inference using vLLM module, is it possible?

**Comment by zspo** [comment]
Absolutely. Currently vllm supports Mistral models. As long as the model is fine-tuned on the Mistral base, it can be used directly. 
<img width="716" alt="image" src="https://github.com/vllm-project/vllm/assets/26846598/219d2931-3586-4c9d-8197-ec1178cb3f5c">
